src/lib/holocene/accordion.svelte

```diff
@@ -59,14 +65,16 @@
         {title}
         <slot name="summary" />
       </h2>
-      <div
-        class="flex flex-row items-center gap-2 pr-2"
-        on:click|stopPropagation
-        on:keyup|stopPropagation
-      >
-        <slot name="action" />
-      </div>
-      <Icon class="m-2" name={open ? 'chevron-up' : 'chevron-down'} />
+      {#if $$slots.default}
+        <div
+          class="flex flex-row items-center gap-2 pr-2"
+          on:click|stopPropagation
+          on:keyup|stopPropagation
+        >
+          <slot name="action" />
```

Comment:
We may still want the action slot even if there is no default slot (there can be links, etc. in it). 

Maybe we move the `#if $$slots.default` conditional to just be for the `Icon`?

Reply:
That's a good point! Thanks Laura!